Content.Client/Implants/UI/ChameleonControllerBoundUserInterface.cs

```diff
@@ -13,6 +13,7 @@ namespace Content.Client.Implants.UI;
 public sealed class ChameleonControllerBoundUserInterface : BoundUserInterface
 {
     [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly ChameleonControllerSystem _chamController = default!;
```

Comment:
I think this fails due to it being a system and not a manager? Might need to set it in the same way `UseDelaySystem` is set.